### PR TITLE
[code] allow ms-toolsai.jupyter to use proposed APIs

### PR DESCRIPTION
**Pull request by akosyakov**
- [x] /werft with-clean-slate-deployment

#### What it does

fix https://github.com/gitpod-io/gitpod/issues/5467

Changes in Gitpod Code: https://github.com/gitpod-io/vscode/commit/6d71890eae337a06260fb41896b443e290af751e

#### How to test

- Start a workspace: https://akosyakov-no-longer-able-to-select-5467.staging.gitpod-dev.com/#https://github.com/DataBooth/public-hello-gitpod
- Install python extension
- Open notebook file
- Press select kernel, you should see some kernels

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **csweichel** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @csweichel` in a comment when ready.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- **[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: a98df1b151fc215d707327098fea829e4594bf74</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5498#issuecomment-911814698" title="LGTM">jeanp413</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **csweichel** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @csweichel` in a comment when ready.

Associated issue: *#5467*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- ~~[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)~~ [jeanp413]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by jeanp413 — APPROVED** [review]

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5498#pullrequestreview-745630106" title="Approved">jeanp413</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **csweichel** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @csweichel` in a comment when ready.

Associated issue: *#5467*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)**
- ~~[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)~~ [jeanp413]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5498#pullrequestreview-745630106" title="Approved">jeanp413</a>*, *<a href="https://github.com/gitpod-io/gitpod/pull/5498#issuecomment-912258518" title="Approved">rl-gitpod</a>*

Associated issue: *#5467*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/gitpod-io/gitpod/blob/main/OWNERS)~~ [rl-gitpod]
- ~~[components/ide/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ide/OWNERS)~~ [jeanp413]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
